src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -456,6 +476,78 @@ protected void setup()
         completeSetup();
     }
 
+    /**
+     * Checks if the data of the local system keyspaces need to be migrated to a different location.
+     *
+     * @throws IOException
+     */
+    private void migrateSystemDataIfNeeded() throws IOException
+    {
+        String importSystemDataFrom = System.getProperty("cassandra.importSystemDataFilesFrom");
```

Comment:
I think that snake case is more common for system properties:
```suggestion
        String importSystemDataFrom = System.getProperty("cassandra.import_system_data_files_from");
```